utils.mk

```diff
@@ -147,6 +147,7 @@ endif
 
 ifeq ($(ARCH), ppc64le)
     override LIBC = gnu
+    override ARCH = powerpc64le
```

Comment:
`powerpc64` ?

Reply:
We are getting the following error:
```
warning: bzip2-sys@0.1.11+1.0.8: Compiler family detection failed due to error: ToolNotFound: Failed to find tool. Is `powerpc-linux-gnu-gcc` installed?
```
e.g. https://github.com/kata-containers/kata-containers/actions/runs/13114872936/job/36586653838?pr=10824

Reply:
Hi @stevenhorsman, I don't have an answer straight away. Will spend some time on this and get back by EOD or tomorrow at the latest.

Reply:
@stevenhorsman, how can I reproduce the error locally? I don't see the rust version change in this PR

Reply:
The full rust version change is in https://github.com/kata-containers/kata-containers/pull/10806, but this PR is mostly just trying to address pre-work and due to the `target_arch` being wrong, but older versions of rust not picking it up then we would just be skipping lots of the power code at the moment, so this should stand alone as is.

Reply:
Able to reproduce the error, should be back with some findings today.

Reply:
@stevenhorsman, I agree with @gkurz on keeping the `gcc_arch` as powerpc64le and while building, the target should be set as `powerpc64le-unknown-linux-gnu` as it is right now.

However, the change needs to be made to the target_arch attribute to `#[cfg(all(target_arch = "powerpc64", target_endian = "little"))]` for moving ahead with rust 1.80.

Does this make sense? 

[This will need code changes accordingly. I've made them but hitting some issues.]

Reply:
Sure, I can try and implement this

Reply:
Ok, I think I have this in place now and have removed a lot of the power64 specific commit, so if you have a chance to review it (the first commit in this PR, currently https://github.com/kata-containers/kata-containers/pull/10824/commits/48ea7db576124fba8a628d2bcd1b9e1923da7c88 until I need to rebase), that would be great. Thanks!

Reply:
Sorry, scrap that - I missed some bits, so need to fix them first!

Reply:
@Amulyam24 - ok, I think I have it back working. All the static-checks on power are passing except the sudo agent test, which also failed on `main`. I ended up splitting out the little endian target pieces into a separate commit, so a review of the first two commits would be good.

Reply:
This change is there to fix the value of `TRIPLE = $(ARCH)-unknown-linux-$(LIBC)`, right ? I'm not sure how this relates to the fixing of `target_arch` in config checks but I won't object :grin:

Reply:
Yeah sorry that commit was a beast of lots of things thrown in when I inherited it and I promptly added a load more generic ppc64 build fixes in and then when I trimmed it down and removed a bunch of problems in there I didn't cultivate it quite enough!